### DISCUSSION TITLE
We can load IPLs but not unload them it seems?

### DISCUSSION THREAD

**Discussion by BUZZARDGTA** [YimMenu/YimMenu#2848] [Ideas]
Quick fix I found is to load a smaller IPL

**Comment by Tessio** [comment]
The system I made together with @YimMenuTheBest for yimmenu, it uses certain patterns, both to load a set of ipls at the same time, and to unload other ipls that can generate conflict in them, besides that when changing from one ipl to another this is unloaded and thus "save memory"? anyway I plan to make an upgrade to this system, but I am terrified that seeing that I use combos and case 1: etc gir489 will tear me to pieces with their comments.

**Reply by Tayipc38** [reply]
Do not be discouraged by his harsh remarks. Try, be corrected and respect feedback.
